### PR TITLE
obelisks protect area from burrows

### PR DESCRIPTION
**Pull request by SolatK**
fixes#3954
Changelog

:cl:
tweak: active obelisks prevents burrow spawn
/:cl:

### CODE REVIEW

File: code/game/objects/structures/burrows.dm [hunk 1]
```diff
@@ -61,6 +61,9 @@
 /obj/structure/burrow/New(var/loc, var/turf/anchor)
 	.=..()
 	all_burrows.Add(src)
+	var/obj/machinery/power/nt_obelisk/obelisk = locate(/obj/machinery/power/nt_obelisk) in range(7, src)
+	if(obelisk || obelisk.active)
+		Destroy()
```
Comment (LethalGhost) [inline code/game/objects/structures/burrows.dm]: Destroy is part of `qdel` system. It shouldn't be called directly.

File: cev_eris.dme [hunk 2]
```diff
@@ -15,57 +15,57 @@
...
+#include "code\__defines\subsystems.dm"
+#include "code\__defines\targeting.dm"
+#include "code\__defines\tick.dm"
+#include "code\__defines\time.dm"
+#include "code\__defines\tools_and_qualities.dm"
+#include "code\__defines\topic.dm"
+#include "code\__defines\turfs.dm"
+#include "code\__defines\unit_testing.dm"
+#include "code\__defines\weapons.dm"
+#include "code\__defines\ZAS.dm"
```
Comment (LethalGhost) [inline cev_eris.dme]: Ugh

Comment (SolatK) [reply cev_eris.dme]: ¯\_(ツ)_/¯

File: code/game/objects/structures/burrows.dm [hunk 3]
```diff
@@ -61,6 +61,9 @@
 /obj/structure/burrow/New(var/loc, var/turf/anchor)
 	.=..()
 	all_burrows.Add(src)
+	var/obj/machinery/power/nt_obelisk/obelisk = locate(/obj/machinery/power/nt_obelisk) in range(7, src)
+	if(obelisk || obelisk.active)
+		qdel(src)
```
Comment (LethalGhost) [inline code/game/objects/structures/burrows.dm]:
```suggestion
		qdel(src)
		return
```
May be like that?

**Pull request merged**